cli/commands/hclvalidate/command.go

```diff
@@ -11,8 +11,8 @@ import (
 const (
 	CommandName = "hclvalidate"
 
-	InvalidFlagName   = "terragrunt-hclvalidate-invalid"
-	InvalidEnvVarName = "TERRAGRUNT_HCLVALIDATE_INVALID"
+	ShowConfigPathFlagName   = "terragrunt-hclvalidate-show-config-path"
+	ShowConfigPathEnvVarName = "TERRAGRUNT_HCLVALIDATE_INVALID"
```

Comment:
If you're already making a breaking change, I think you should rename this to `TERRAGRUNT_HCLVALIDATE_SHOW_CONFIG_PATH` or `TERRAGRUNT_HCLVALIDATE_SHOW_INVALID_CONFIGS`

Reply:
Indeed, I missed it. Thank you.